.vscode/launch.json

```diff
@@ -20,6 +20,22 @@
                 }
             },
             "args": [],
-        }
+        },
+        {
+            "type": "node-terminal",
+            "name": "Debug Current TS File (npx tsx)",
+            "request": "launch",
+            "command": "npx tsx -- ${fileBasenameNoExtension}",
+            "cwd": "${fileDirname}",
+            "envFile": "${workspaceFolder}/front/.env.local",
```

Comment:
IDK what we would want to do with that env file but it was handy for me locally. A bit messy to have all vars there though